public/javascripts/SVValidate/src/Main.js

```diff
@@ -241,7 +246,7 @@ function Main (param) {
             html: true
         });
 
-        const labelType = param.labelList[0].getAuditProperty('labelType');
+        constlabelType = param.labelList[0].getAuditProperty('labelType');
```

Comment:
Please double check the changes when you're creating the PR. This is an accidental change you made. For some reason, the outcome seems to be that keyboard shortcuts don't work during the first mission? Not what I expected! But either way, it's an obvious typo that should have been caught beforehand.